### PR TITLE
trim location coordinates from gtfs stop points

### PR DESCRIPTION
**Pull request by nberard**
pl_warsaw data set contains spaces before certain longitudes

### CODE REVIEW

File: src/utils.rs [hunk 1]
```diff
@@ -113,6 +113,18 @@ where
...
 
+pub fn de_location_trim_with_default<'de, D>(deserializer: D) -> Result<f64, D::Error>
+    where
+        D: ::serde::Deserializer<'de>,
+{
+    use serde::Deserialize;
+    let s = String::deserialize(deserializer)?;
+    Ok(s.trim().parse::<f64>().unwrap_or_else(|e| {
+        error!("{}", e);
+        0.00
```
Comment (kinnou02) [inline src/utils.rs]: Not a fan of putting stop_point on the null island, but if it's ok for @prhod I'am ok too

File: src/gtfs/mod.rs [hunk 2]
```diff
@@ -91,9 +91,9 @@ struct Stop {
     name: String,
     #[serde(default, rename = "stop_desc")]
     desc: String,
-    #[serde(rename = "stop_lon")]
+    #[serde(rename = "stop_lon", deserialize_with = "de_location_trim_with_default")]
```
Comment (datanel) [inline src/gtfs/mod.rs]: you could use https://docs.rs/csv/1.0.2/csv/struct.ReaderBuilder.html#method.trim

Comment (nberard) [reply src/gtfs/mod.rs]: agreed with u if we can trim all fields, what do u think @kinnou02?

File: src/utils.rs (hunk 1 above)
Comment (nberard) [reply src/utils.rs]: not a fan too but he made me do it :p

Comment (prhod) [reply src/utils.rs]: @kinnou02  I'm open to any other solution that enable conversion (without panick)  and if possible not much refacto ^^ 
Using a blank value doesn't seem to be a good solution :/

File: src/gtfs/mod.rs (hunk 2 above)
Comment (kinnou02) [reply src/gtfs/mod.rs]: good for me

Comment (prhod) [reply src/gtfs/mod.rs]: trimming both content and header seems to be a good idea, we know some GTFS have spaces in col names

**Pull request merged**